airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/dev_assets.py

```diff
@@ -0,0 +1,166 @@
+import yaml
+from dagster import Output, asset
+
+"""
+NOTE: This file is temporary and will be removed once we have metadata files checked into source control.
+
+TODO (ben): Remove this file once we have metadata files checked into source control.
+"""
+
+GROUP_NAME = "dev"
+
+OVERRIDES = {
```

Comment:
This is all the info we added which was missing in the loom, right?

Reply:
Oooh I realized I should add a comment here.

But yes! These are all the manual catalog adjustments we must make for all generated metadata files to pass validation